src/schemas/land.schema.ts

```diff
@@ -0,0 +1,20 @@
+import { object, string, number } from 'yup';
+
+export const createLandSchema = object({
+    body: object({
+        address: string(),
+        commune: string()
+            .required(`Commune name is required`)
+            .min(2, `The commune name is to short - should be 2 chars minium`)
+            .max(75, `The commune name is to long - should be 75 chars maximum`)
+            .matches(/^[a-zA-Z0-9_.-]*$/, `Commune name can only contain Latin letters.`),
+        postalCode: number()
```

Comment:
Can you change this number() by string() because actually we can find some postal code with letters.